src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
You misunderstood the previous comment a bit. No need to move the index calculation here. Just the Random object itself.